Comments/ClientApp/public/index.html

```diff
@@ -74,37 +74,34 @@
 	<iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NR4W7NR"
 			height="0" width="0" style="display:none;visibility:hidden"></iframe>
 </noscript>
-<!-- End Google Tag Manager (noscript) -->
-
-<div id="root"></div>
 
-<!-- Jump to main content (end of tophat) -->
-<script>
-	document.onTophatReady = function () {
-		document
-				.querySelector("#skip-to-content")
-				.addEventListener("click", function (e) {
-							e.preventDefault();
-							var tophatEnd = document.querySelector("#tophat-end");
-							tophatEnd.setAttribute("tabindex", "-1");
-							tophatEnd.focus();
-						}
-				);
+<!-- Global nav -->
+<script data-accounts-env="%REACT_APP_ACCOUNTS_ENVIRONMENT%" id="globalnavScript">
+	
+	var global_nav_config = {
+		header: {
+			skipLinkId: "content-start",
+			auth: {
+				environment: document.getElementById("globalnavScript").getAttribute("data-accounts-env")
+			}
+		},
+		footer: false
 	};
 </script>
 
-<!-- Tophat -->
-<script src="//cdn.nice.org.uk/V3.1/Scripts/nice/NICE.TopHat.min.js"
-		data-environment="Live"
-		data-search="/search?q=%term"
-		data-typeaheadtype="remote"
-		data-typeaheadsource="/autocomplete?%query"
-		data-search-placeholder="Search NICE..."
-		data-cookie="true"
-		async></script>
+<!--[if lt IE 9]>
+	<script src="%REACT_APP_GLOBAL_NAV_SCRIPT_IE8%"></script>
+<![endif]-->
+<script src="%REACT_APP_GLOBAL_NAV_SCRIPT%"></script>
```

Comment:
Global nav doesn't wait for document ready (maybe it should?) but the script include would need to be after the div you've created, otherwise it won't find it, so will create its own